gamemodes/terrortown/gamemode/client/cl_player_ext.lua

```diff
@@ -246,8 +246,8 @@ net.Receive("TTT2RevivalUpdate_IsReviving", function()
 	LocalPlayer().isReviving = net.ReadBool()
 end)
 
-net.Receive("TTT2RevivalUpdate_IsBlockingRevival", function()
-	LocalPlayer().isBlockingRevival = net.ReadBool()
+net.Receive("TTT2RevivalUpdate_RevivalBlockMode", function()
+	LocalPlayer().revivalBlockMode = net.ReadReadUInt(2)
```

Comment:
Pretty sure this is a typo: "net.ReadReadUInt(2)" should be "net.ReadUInt(2)".

Reply:
huh, strange that I did not notice an error